about.html

```diff
@@ -0,0 +1,235 @@
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="bottom-navbar ">
+                    <div class="container-fluid flex-adjust-center">
+                        <div class="left-side">
+
+                            <a href="index.html">
+                                <img src="assets/img/logo-400.jpg" alt="logo-header" class=" d-none d-md-block">
+                            </a>
+                        </div>
+                        <div class="center-list d-none d-md-block">
+                            <ul class="list-unstyled flex-adjust-center">
+                                <li>
+                                    <a href="about.html">
+                                        <span class="text-secondary-color font-weight-bold">About</span>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#active">
+                                        <span class="text-secondary-color font-weight-bold">Activites</span>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#partners">
+                                        <span class="text-secondary-color font-weight-bold">Partners</span>
+                                    </a>
+                                </li>
+                            </ul>
+
+                        </div>
+                        <button type="button" class=" campaign-button d-none d-md-block">
+                            Plan</button>
+                    </div>
+                </div>
+            </section>
+        </nav>
+        <div class="behind-navbar"></div>
+
+        <div class="opacity">
+
+            <button type="button" class="ham">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+                </svg>
+            </button>
+        </div>
+
+        </div>
+
+        <div class="overlayy"></div>
+        <nav class="sideNav">
+            <button type="button" class="menuX">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                </svg>
+            </button>
+            <ul>
+                <li><a class="menuLink" href="index.html">Home</a></li>
+                <li><a class="menuLink" href="#projects-container">Contact</a></li>
+                <li><a class="menuLink" href="#about-container">Sponsort</a></li>
+                <li><a class="menuLink main-button" href="#footer">Buy Tickets</a></li>
+            </ul>
```

Comment:
- [ ] Please add a link to the about page so the users can navigate through your website easier.